apps/amiapp_flutter/android/app/src/main/AndroidManifest.xml

```diff
@@ -59,6 +59,15 @@
                 <data android:pathPrefix="/dashboard" />
             </intent-filter>
         </activity>
+        <!-- Register Customer.io Firebase Messaging Service as we want our SDK to handle all push
+             notifications -->
+        <service
+            android:name="io.customer.messagingpush.CustomerIOFirebaseMessagingService"
```

Comment:
The notifications were not working for Android apps, so I added Option 2 so our SDK handles everything in it.

Reply:
Great thanks, can you also make sure Android devices are well registered with this update ?

https://github.com/customerio/customerio-flutter/issues/113

Reply:
Hi @mrehan27 I've just tested [version 1.3.2](https://github.com/customerio/customerio-flutter/releases/tag/1.3.2) & it does seem to have resolved the issue https://github.com/customerio/customerio-flutter/issues/113 🙂

Thank you 🙏

Reply:
Glad to hear. Thanks for confirming 👍🏻